internal/testing/cluster_test.go

```diff
@@ -368,6 +368,46 @@ func TestUploadDownloadBasic(t *testing.T) {
 			t.Fatal("unexpected")
 		}
 	}
+
+	// fetch the config
+	cfg, err := cluster.Autopilot.Config()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// shut down the autopilot to prevent it from reinstating the contract set
+	if err := cluster.ShutdownAutopilot(context.Background()); err != nil {
```

Comment:
Might be easier to specify a custom contract set and update the settings to use that set instead of "autopilot".